src/LiveComponent/doc/index.rst

```diff
@@ -2255,26 +2259,88 @@ real component once the page has loaded.
     page load, but it isn't rendered. So keep your heavy work to methods in
     your component (e.g. ``getProducts()``) that are only called when rendering.
 
-To add some loading text before the real component is loaded, use the
-``loading-template`` option to point to a template:
+To change the initial tag from a ``div`` to something else, use the ``loading-tag`` option:
 
 .. code-block:: twig
 
+    {{ component('SomeHeavyComponent', { defer: true, loading-tag: 'span' }) }}
+
+To add some loading content before the real component is loaded, you can use the
+``loading-template`` option to point to a template:
+
+.. code-block:: html+twig
+
+    {# With the HTML syntax #}
+    <twig:SomeHeavyComponent defer loading-template="spinning-wheel.html.twig" />
+
+    {# With the component function #}
     {{ component('SomeHeavyComponent', { defer: true, loading-template: 'spinning-wheel.html.twig' }) }}
 
 Or override the ``loadingContent`` block:
 
-.. code-block:: twig
+.. code-block:: html+twig
 
+    {# With the HTML syntax #}
+    <twig:SomeHeavyComponent defer>
+        <twig:block name="loadingContent">Custom Loading Content...</twig:block>
+    </twig:SomeHeavyComponent>
+
+    {# With the component tag #}
     {% component SomeHeavyComponent with { defer: true } %}
         {% block loadingContent %}Loading...{% endblock %}
     {% endcomponent %}
 
-To change the initial tag from a ``div`` to something else, use the ``loading-tag`` option:
+Both ``loading-template`` and ``loadingContent`` can be used in the template
+_calling_ the component. But if want to set this content only once, you can add a
+``placeholder`` macro directly in the component template.
```

Comment:
macro called placeholder

that makes it more clear that placeholder with name vs placeholder is an adjective

Reply:
"named" placeholder ?